### DISCUSSION TITLE
Dim led's

### DISCUSSION THREAD

**Discussion by kiasho** [awawa-dev/HyperHDR#632] [Q&A]
I have hyperhdr 18 running on a rpi4 and the led's connected via an esp8266 it all appears to be working fine but even though my power supply is a 5v 15amp one thats powering the rpi and the led strip and the esp is being powered from the rpi but the led's seem dim compared with many ive seen on youtube.
Any ideas why

**Comment by awawa-dev** [comment]
Update to v19

**Reply by kiasho** [reply]
updated and the led's look exactly the same

**Reply by awawa-dev** [reply]
LED driver?

**Reply by kiasho** [reply]
if you mean what is wled on it's a esp8266

**Reply by awawa-dev** [reply]
WLED configuration is on your own, there are many settings that could affect final brightness for example: the unfortunate WLED power limiter. You can also reset HyperHDR settings to default settings to make sure that changes made, e.g. for color calibration in the processing tab, do not cause this.

**Reply by kiasho** [reply]
ok ill try that ty
